client/my-sites/themes/thanks-modal.scss

```diff
@@ -53,10 +53,11 @@
 		}
 	}
 
-	.thanks-modal__gutenberg-prompt {
+	&-fse-notice {
+		margin: 1rem auto;
 		color: var( --color-text-subtle );
-		font-size: 14px;
-		margin-top: 12px;
+		font-size: 0.875rem;
```

Comment:
Could we use a font prescribed by `packages/typography/styles/variables.scss` (which already appears to use rem units)?

https://github.com/Automattic/wp-calypso/blob/a67e5be134d785af8dbd2612270f4290f36f662d/packages/typography/styles/variables.scss#L27-L48